src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -141,81 +139,28 @@ private async Task WriteJsonInstance(Stream stream, IExceptionInstance instance,
             }
 
-            // Make sure dotnet-monitor is self-consistent with other features that print type and stack information.
-            // For example, the stacks and exceptions features should print structured stack traces exactly the same way.
-            // CONSIDER: Investigate if other tools have "standard" formats for printing structured stacks and exceptions.
-            await using (Utf8JsonWriter writer = new(stream, new JsonWriterOptions() { Indented = false }))
+            Models.ExceptionInstance model = new()
             {
-                writer.WriteStartObject();
-                writer.WriteNumber("id", instance.Id);
-                // Writes the timestamp in ISO 8601 format
-                writer.WriteString("timestamp", instance.Timestamp);
-                writer.WriteString("typeName", instance.TypeName);
-                writer.WriteString("moduleName", instance.ModuleName);
-                writer.WriteString("message", instance.Message);
-
-                if (IncludeActivityId(instance))
-                {
-                    writer.WriteStartObject("activity");
-                    writer.WriteString("id", instance.ActivityId);
-                    writer.WriteString("idFormat", instance.ActivityIdFormat.ToString("G"));
-                    writer.WriteEndObject();
-                }
-
-                writer.WriteStartArray("innerExceptions");
-                foreach (ulong innerExceptionId in instance.InnerExceptionIds)
-                {
-                    writer.WriteStartObject();
-                    writer.WriteNumber("id", innerExceptionId);
-                    writer.WriteEndObject();
-                }
-                writer.WriteEndArray();
-
-                if (null != instance.CallStack)
+                Id = instance.Id,
+                Timestamp = instance.Timestamp,
+                TypeName = instance.TypeName,
+                ModuleName = instance.ModuleName,
+                Message = instance.Message,
+                InnerExceptionIds = Array.ConvertAll(instance.InnerExceptionIds, id => (InnerExceptionId)id),
+                CallStack = instance.CallStack,
+            };
+
+            if (IncludeActivityId(instance))
+            {
+                model.Activity = new()
                 {
-                    writer.WriteStartObject("stack");
-                    writer.WriteNumber("threadId", instance.CallStack.ThreadId);
-                    writer.WriteString("threadName", instance.CallStack.ThreadName);
-
-                    writer.WriteStartArray("frames");
-
-                    StringBuilder builder = new StringBuilder();
-
-                    foreach (var frame in instance.CallStack.Frames)
-                    {
-                        writer.WriteStartObject();
-
-                        string assembledMethodName = frame.MethodName;
-                        if (frame.FullGenericArgTypes.Count > 0)
```

Comment:
It looks like `/stacks` and `/exceptions` diverge here. Exception frames include generic arg types while stacks don't... I'll work on restoring this behavior for `/exceptions`